<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptRosterMain</class>
 <widget class="QWidget" name="OptRosterMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptApplicationUI</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QCheckBox" name="ck_alwaysOnTop">
     <property name="text">
      <string>Roster window always on top</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_autoRosterSize">
     <property name="text">
      <string>Auto-resize contact list</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_useleft">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Use left-click for menu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_showMenubar">
     <property name="text">
      <string>Show menubar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_disableScrollbar">
     <property name="toolTip">
      <string>Disables scrollbar in the application window.</string>
     </property>
     <property name="text">
      <string>Disable scrollbar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_roster_avatar">
     <property name="toolTip">
      <string>Enables avatar frame at the top of roster</string>
     </property>
     <property name="text">
      <string>Enable roster avatar frame</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Autohide roster after</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_hideInterval">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>seconds (0 - never)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_docklet">
     <property name="title">
      <string>Icons</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QCheckBox" name="ck_showGeoLocationIcons">
          <property name="toolTip">
           <string>Show geolocation in roster</string>
          </property>
          <property name="text">
           <string>GeoLocation</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="ck_showMoodIcons">
          <property name="toolTip">
           <string>Show moods in roster</string>
          </property>
          <property name="text">
           <string>Moods</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="ck_showAvatarIcons">
          <property name="toolTip">
           <string>Show avatars in roster</string>
          </property>
          <property name="text">
           <string>Avatars</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="ck_showActivityIcons">
          <property name="toolTip">
           <string>Show activities in roster</string>
          </property>
          <property name="text">
           <string>Activities</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="ck_showTuneIcons">
          <property name="toolTip">
           <string>Show icon when contact is listening to music</string>
          </property>
          <property name="text">
           <string>Now playing</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="ck_showClientIcons">
          <property name="toolTip">
           <string>Show icon of user client</string>
          </property>
          <property name="text">
           <string>Clients</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="cb_showAllClientIcons">
          <item>
           <property name="text">
            <string>all client icons</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>only highest priority client icon</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_avatarsSettings">
     <property name="title">
      <string>Avatar settings</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbl_avatarsSize">
          <property name="text">
           <string>Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_avatarsSize">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string>pt</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>64</number>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_avatarsRadius">
          <property name="text">
           <string>Radius of corners</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_avatarsRadius">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string>pt</string>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ck_alwaysOnTop</tabstop>
  <tabstop>ck_autoRosterSize</tabstop>
  <tabstop>ck_useleft</tabstop>
  <tabstop>ck_showMenubar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
